packages/react-avatar/src/components/Avatar/index.ts

```diff
@@ -1,4 +1,5 @@
-export * from './Avatar.types';
+// Explicit exports to ommit AvatarCommon
+export type { AvatarSlots, AvatarNamedColor, AvatarProps, AvatarState } from './Avatar.types';
```

Comment:
nit: Order these alphabetically
```suggestion
export type { AvatarNamedColor, AvatarProps, AvatarSlots, AvatarState } from './Avatar.types';
```